rcon/extended_commands.py

```diff
@@ -284,6 +285,15 @@ def do_perma_ban(self, player, reason):
         with invalidates(self.get_players, self.get_perma_bans):
             return super().do_perma_ban(player, reason)
 
+    def set_randomMessages(self, bool_str, log_info=True):
```

Comment:
You added it so that it's exposed in the http API? 
While it's convenient to add it like that maybe we should make it leave in its own module. This classes is already super lengthy,